lab/AnimatedTangle/patterns/coral.js

```diff
@@ -0,0 +1,172 @@
+import { Oklch } from "../../lablib/Oklch.js";
+import { Direction } from "../../../pga2d/Direction.js";
+import { Point } from "../../../pga2d/Point.js";
+import { Color } from "../../../sketchlib/Color.js";
+import { CirclePrimitive } from "../../../sketchlib/primitives/CirclePrimitive.js";
+import { InvMask, Mask } from "../../../sketchlib/primitives/ClipMask.js";
+import { RectPrimitive } from "../../../sketchlib/primitives/RectPrimitive.js";
+import { group, style } from "../../../sketchlib/primitives/shorthand.js";
+import { VectorTangle } from "../../../sketchlib/primitives/VectorTangle.js";
+import { Style } from "../../../sketchlib/Style.js";
+import { CoralNode, CoralTree } from "../CoralTree.js";
+import { AnimatedStripes, make_stripes } from "./stripes.js";
+
+const RADIUS_BIG = 25;
+const RADIUS_SMALL = RADIUS_BIG / 2;
+
+const TREE = new CoralTree(
```

Comment:
Make a note: To get the branches of the tree to sway a bit, I'm going to need to store nodes in variables so I can update the positions and re-render the tree.

Best to wrap this in a class like I've done with the other sketches.

This will also be important when I add decoration to the coral